.github/workflows/go.yml

```diff
@@ -0,0 +1,200 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - release/*
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+jobs:
+  check:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go:
+          - version: 1.15.x
+            may-fail: false
+          - version: 1.16.x
+            may-fail: true
+          # TODO enable when 1.15.x is removed
+#          - version: tip
+#            may-fail: true
+
+    continue-on-error: ${{ matrix.go.may-fail }}
+
+    env:
+      GOPATH: ${{ github.workspace }}
+      WORKING_DIR: ${{ github.workspace }}/src/github.com/percona/pmm-agent
+      GO111MODULE: auto
```

Comment:
I think it is better to use `off` explicitly, as what `auto` means changes a bit between Go versions

Reply:
If it's off Go 1.16 is failing

Reply:
done